src/org/parosproxy/paros/network/SSLConnector.java

```diff
@@ -108,7 +109,9 @@
 		SECURITY_PROTOCOL_TLS_V1,
 		SECURITY_PROTOCOL_TLS_V1_1,
 		SECURITY_PROTOCOL_TLS_V1_2,
-		SECURITY_PROTOCOL_TLS_V1_3 };
+		// Disable TLS 1.3 by default as it currently fails with Java 11
+		// SECURITY_PROTOCOL_TLS_V1_3 

```

Comment:
This could be disabled just on the client side, the server side seems to be working fine.